### ISSUE TITLE
Incompatible with FF 36.0.4. 

### ISSUE THREAD

**Issue by awalkonthebeach** [ithinc/tabutils#155]
No comment.

**Comment by yfdyh000** [comment]
I use it works normal.

Please provide more information, such as: OS's version, TU's version, error messages in Browser Console, how to reproduce, screenshots.

**Comment by yfdyh000** [comment]
Sorry, I was wrong about the project name.
Use the https://addons.mozilla.org/firefox/addon/tab-utilities-fixed/.

**Comment by awalkonthebeach** [comment]
yfdyh000, 

So you have taken over the project from the original author?

If that is the case, I suggest that you post an announcement on the original TU page, and provide a link to your "TU fixed" extension .

https://addons.mozilla.org/en-US/firefox/addon/tab-utilities/?src=search

Thank you for keeping this project alive!

**Comment by yfdyh000** [comment]
No, the original author is no audio, so I created a Fork (non-perfect).

Being tracked at: https://bugzilla.mozilla.org/show_bug.cgi?id=1138986
